bin/main.rb

```diff
@@ -0,0 +1,3 @@
+# ! /usr/bin/env ruby
```

Comment:
- [ ] the expression has a space between `#` and `!`, which makes the script not work as expected. please remove the space or use the next expression `#! /usr/bin/env ruby`